docs/sponsor/call-for-machines.md

```diff
@@ -0,0 +1,42 @@
+# Call for Machines
+
+As mentioned in several research works, the power consumption behavior can be varied by several factors.
+Using the right power model to predict power consumption on the machine that has no power meter is critical the precision of the reported values.
+
+We are seeking machines with the following characteristics, commonly used in cloud computing environments but not limited to:
+
+## Processor
+- Intel: Cascade Lake, Skylake (or newer)
+- Ampere: Altra, AmpereOne (or newer)
+- AMD: Opteron, EPYC (or newer)
+- Apple: M1, M2 (or newer)
+
+## Accelerator Chips
+- GPU: NVIDIA A100, V100, or other high-performance models for AI/ML workloads
```

Comment:
AMD GPU, and also NVIDIA Grace Hopper.

Reply:
Since GPU resources are a bit expensive, we are happy to have them for a time windowed reservation.

Reply:
@rootfs I removed AMD GPU from the list since it seems that we are not supporting it in Kepler yet.

Instead, I added the following note:

> Please note that power source modules for other accelerator cards, such as AMD GPUs, Google TPUs, and IBM AIUs, are not yet supported by Kepler.
> Contributions to support these devices are always welcome within the Kepler community.

I added the expected use with a new section.